Dockerfile

```diff
@@ -1,17 +1,16 @@
-FROM openjdk:8-jre-alpine
+FROM alpine:3.12
+
 MAINTAINER OpenZipkin "http://zipkin.io/"
 
-# Add edge repo, needed for tools downstream like runit
+# Default to UTF-8 file.encoding
+ENV LANG C.UTF-8
+
+# Add edge repo, needed for latest JRE and tools downstream like runit
 RUN echo "http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories
-# Avoid warning: This apk-tools is OLD!
-RUN apk add --upgrade --no-cache apk-tools
 
-# Setup curl and bash for convenience as all derivative images use it
-RUN apk add --update --no-cache curl bash apk-tools
```

Comment:
bash wasn't needed. cassandra had a script using it, but only because we added shebang /bin/bash :P